client/src/html/driver_display.html

```diff
@@ -0,0 +1,55 @@
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>UWMidsun Driver Display</title>  
+    <meta name="description" content="UWMidsun Driver Display">
+    <meta name="author" content="UWMidsun">
+    <link href="../fonts/OpenSans-Regular.ttf" rel="stylesheet">
+    <link href="../css/stylesheet.css" rel="stylesheet">
+    <link href="../css/plottable.css" rel="stylesheet">
+    <script src="../static/d3.js"></script>
+    <script src="../static/plottable.js"></script>
+  </head>
+
+
+  <body style="width:100%; height:100%">
```

Comment:
Personally not a fan of inline ``style``. Can we move this to the ``scss`` file?